### ISSUE TITLE
Auto focus chat entry field when chat is initiated

### ISSUE THREAD

**Issue by lmbrt** [dm0-/window_merge#9]
Currently if you click on a contact to initiate the chat, the buddy list is still focused.

Seems like it should move the focus to the conversation window's entry field.

**Comment by dm0-** [comment]
Thanks for reporting.  I have pushed a change that corrects this for me.  Let me know if that doesn't fix it.

**Comment by lmbrt** [comment]
Thank you for your quick response.  Unfortunately this does not work if I initiate the chat...

i.e. I double click on a contact, then begin typing.  The buddy list is still the focus...

**Comment by dm0-** [comment]
Sorry, I noticed that change only corrected the issue with IMs, not chats.  It seems that the "conversation-created" signal was being called before GTK processed a focus event from clicking on the Buddy List, which stole back the focus immediately.  I've pushed a slight change that runs though queued events before focusing the conversation entry field which now appears to fix the issue for me--again.

(I don't particularly like this idea of processing queued events from within a signal handler, so consider this a temporary fix.  I will try to stabilize and push a rewrite of the Pidgin signal handlers shortly.)

I should also note that this force-focus only takes effect when opening the first conversation in the window, since that is the only instance where I could reproduce the issue.  Let me know if this is not the case for you or if the issue persists.
